dart/lib/src/protocol/sentry_log_attribute.dart

```diff
@@ -0,0 +1,29 @@
+class SentryLogAttribute {
+  final dynamic value;
+  final String type;
+
+  const SentryLogAttribute(this.value, this.type);
+
+  factory SentryLogAttribute.string(String value) {
+    return SentryLogAttribute(value, 'string');
+  }
+
+  factory SentryLogAttribute.boolean(bool value) {
+    return SentryLogAttribute(value, 'boolean');
+  }
+
+  factory SentryLogAttribute.integer(int value) {
+    return SentryLogAttribute(value, 'integer');
+  }
+
+  factory SentryLogAttribute.double(double value) {
+    return SentryLogAttribute(value, 'double');
+  }
+
+  Map<String, dynamic> toJson() {
+    return {
+      'value': value,
+      'type': type,
+    };
+  }
+}
```

Comment:
can you add a quick comment that in the future arrays will be supported (string[], boolean[], integer[], double[]) pls